src/ngraph/op/fused/depth_to_space.hpp

```diff
@@ -56,25 +57,43 @@ namespace ngraph
                 /// \param block_size The size of the block of values to be moved
                 DepthToSpace(const Output<Node>& data,
                              const DepthToSpaceMode& mode,
-                             std::size_t block_size = 1);
+                             size_t block_size = 1);
```

Comment:
can it be changed to int64_t?

Reply:
Let's do that separately. But we can undo this particular change which was a partial backing out of a failed attempt to change to int64_t here.